app/src/main/java/org/desarrolladorslp/teckersapp/service/AppCredentialsInterceptor.kt

```diff
@@ -0,0 +1,28 @@
+package org.desarrolladorslp.teckersapp.service
+
+import android.util.Base64
+import okhttp3.Interceptor
+import okhttp3.Response
+import org.desarrolladorslp.teckersapp.BuildConfig
+
+
+class AppCredentialsInterceptor() : Interceptor {
+
+    override fun intercept(chain: Interceptor.Chain): Response {
+
+        val appCredentials  = BuildConfig.user +":"+ BuildConfig.password
+        val charset = Charsets.UTF_8
+        val byteArray = appCredentials.toByteArray(charset)
+        val base64 = Base64.encodeToString(byteArray, Base64.DEFAULT)
+        val value = base64.replaceFirst("\n", "")
```

Comment:
Is base64 returning a string with a line feed at the end?

Reply:
yeah that´s why i have to take it out

Reply:
interesting, you could use .trim() instead ... it will remove any trailing spaces